bindings/csharp/Keystone.Net/NativeInterop.cs

```diff
@@ -8,29 +9,34 @@ namespace Keystone
     /// </summary>
     internal class NativeInterop
     {
-        // This shouldn't be needed, even on Windows
-        // /// <summary>
-        // /// Taken from: http://stackoverflow.com/questions/10852634/using-a-32bit-or-64bit-dll-in-c-sharp-dllimport
-        // /// </summary>
-        // static NativeInterop()
-        // {
-        //     var myPath = new Uri(typeof(NativeInterop).Assembly.CodeBase).LocalPath;
-        //     var myFolder = Path.GetDirectoryName(myPath);
+        /// <summary>
+        /// Load the appropriate dynamic-link library, according the architecture of the running application.
+        /// </summary>
+        /// <remarks>
+        /// Taken from: http://stackoverflow.com/questions/10852634/using-a-32bit-or-64bit-dll-in-c-sharp-dllimport
+        /// </remarks>
+        static NativeInterop()
```

Comment:
I'm kinda curious about this: Is it ever needed? I removed it from the original commit because after tests in different environments I never ended up needing this.